packages/types/deno.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "@evex/linejs-types",
+	"version": "0.0.0",
+	"license": "MIT",
+	"exports": {
+		".": "./line_types.ts",
+		"./entities": "./entities/index.ts",
+		"./read": "./read.ts",
+		"./declares": "./declares.ts",
+		"./thrift": "./thrift.ts",
+		"./parser": "./parser.ts",
+		"./write": "./write.ts"
```

Comment:
These are internal types and functions.
So, you don't need include it in `linejs-types`.
Please transfer to `/packages/linejs/libs/thrift` from here.